Lib/glyphsLib/builder/__init__.py

```diff
@@ -12,6 +12,177 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .glyphs import to_glyphs
+import logging
+
+from glyphsLib import classes
+from glyphsLib.util import clear_data
+import defcon
+
+from .context import UFOToGlyphsContext, GlyphsToUFOContext
+from .constants import PUBLIC_PREFIX, GLYPHS_PREFIX
+
+# Existing
+from .kerning import add_glyph_to_groups, add_groups_to_ufo, load_kerning
+from .anchors import propagate_font_anchors, propagate_glyph_anchors
```

Comment:
`propagate_glyph_anchors` is imported but not used here. I think it's a private function only used by `propagate_font_anchros`.